### ISSUE TITLE
Not compatible with python 3.10

### ISSUE THREAD

**Issue by DivineSentry** [bamthomas/aioimaplib#75]
mainly getting errors due to python 3.10 no using a loop anymore, sorry not able to provide more details atm

**Comment by bamthomas** [comment]
Thanks I'll have a look

**Comment by therion23** [comment]
Looks like this:

```
  File "/opt/homeassistant/lib/python3.10/site-packages/aioimaplib/aioimaplib.py", line 326, in __init__
    self._idle_event = asyncio.Event(loop=loop)
  File "/usr/lib/python3.10/asyncio/locks.py", line 168, in __init__
    super().__init__(loop=loop)
  File "/usr/lib/python3.10/asyncio/mixins.py", line 17, in __init__
    raise TypeError(
TypeError: As of 3.10, the *loop* parameter was removed from Event() since it is no longer necessary
```

Hope it helps any.

**Comment by DivineSentry** [comment]
fixed by #76
